misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormatterUtils.java

```diff
@@ -79,4 +85,55 @@ static void sortImportDeclarations(List<ImportDeclarationNode> importDeclaration
                         node2.moduleName().stream().map(node -> node.toString().trim()).collect(Collectors.joining()))
                 .toComparison());
     }
+
+    /**
+     * Swap leading minutiae of the first import in the code and the first import when sorted.
+     *
+     * @param firstImport First ImportDeclarationNode in user code
+     * @param imports     Sorted formatted ImportDeclarationNode nodes
+     */
+    static void swapLeadingMinutiae(ImportDeclarationNode firstImport, List<ImportDeclarationNode> imports) {
```

Comment:
```suggestion
    static void swapLeadingMinutiae(ImportDeclarationNode firstImportNode, List<ImportDeclarationNode> importNodes) {
```
Here also, and all other relevant places.